Sources/Fullscreen/AddressView/AddressView.swift

```diff
@@ -110,16 +110,36 @@ public class AddressView: UIView {
         mapView.setRegion(region, animated: animated)
     }
 
+    private var tileOverlay: MKTileOverlay?
+
+    public func addTileOverlay(tileOverlay: MKTileOverlay) {
```

Comment:
Could this be named `updateTileOverlay(_ tileOverlay: MKTileOverlay)`? Since we're not adding, but removing and setting a new overlay.

Reply:
Or perhaps `configure(tileOverlay: MKTileOverlay)`. Then we could use the same naming conventions on the other `addX`-methods.